mne/viz/ica.py

```diff
@@ -400,7 +400,7 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
     exclude_labels = list()
     for ii in picks:
         if ii in exclude:
-            line_label = 'ICA %03d' % (ii + 1)
```

Comment:
Okay, I mainly use this plotting function as part of the ICA artefact rejection routine - in order to validate the choice of components for rejection, which I do by cross-checking the components between the different plotting modalities (topos, scores, clean, sources, etc.). And in this case, these latent sources refer to ICA components rather than channels, right? Or am I missing something obvious?
I mean, Isn't there a 1-to-1 correspondence between the reconstructed latent source and the given component? At least, I can't figure out what "channels" it would be referring to, and thus how to use the 1-indexing usefully.
I'm sensing you don't want to go with just "ICA #%d" % i, but I'm afraid that's my best suggestion for now.